web-services/query/src/main/java/datawave/webservice/query/logic/QueryLogic.java

```diff
@@ -65,6 +71,23 @@
      */
     public QueryLogicTransformer getTransformer(Query settings);
     
+    default public String getResponseClass() throws QueryException {
+        Query q = new QueryImpl();
```

Comment:
Would it be better to pass a Query in as a parameter to getResponseClass? There are no usages of getResponseClass in this PR that do not have a Query instance available for use, and after I changed it, all the tests still pass.

Reply:
Pushed this change. Good idea on this, now `QueryLogic`s have the option of returning different response classes based on the query passed.

Reply:
After discussing with you, it seems that the better argument to pass is the QueryLogicTransformer, because that is what creates the response object.